psi4/src/psi4/cc/amps.cc

```diff
@@ -59,8 +62,8 @@ std::map<std::string, SharedMatrix> CCEnergyWavefunction::get_amplitudes() {
             cachelist = cacheprep_rhf(options_.get_int("CACHELEVEL"), cachefiles.data());
             spaces = {DPDMOSpace{'o', "ijkl", occpi_}, DPDMOSpace{'v', "abcd", virtpi_}};
         } else /*UHF*/{
-            Dimension aoccpi_ = nalphapi_;
-            Dimension boccpi_ = nbetapi_;
+            Dimension aoccpi_ = nalphapi_ - frzcpi_;
+            Dimension boccpi_ = nbetapi_ - frzcpi_;
```

Comment:
good catch! do RHF and ROHF need modification?

Reply:
I would say yes! Will push that.

Reply:
Yes, very good catch!